go.mod

```diff
@@ -97,12 +97,15 @@ require (
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.18.1 // indirect
 	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83 // indirect
-	golang.org/x/net v0.0.0-20210610132358-84b48f89b13b // indirect
+	golang.org/x/mod v0.5.1 // indirect
```

Comment:
Why all these `go.mod` changes? The PR is not changing any imports nor bumping dependencies. For clarity I would revert dependencies changes.

Reply:
I've just reverted the changes.